content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/clientlibs/editor/js/image.js

```diff
@@ -108,35 +108,27 @@
                     polarisRepositoryId = JSON.parse(cfg).repositoryId;
                     if (polarisRepositoryId) {
                         isPolarisEnabled = true;
-                        remoteFileReference = $cqFileUpload.find("input[name='./fileReference']").val();
+                        remoteFileReferencesArray = $cqFileUpload.find("[data-cq-fileupload-parameter='filereference']");
                     }
                 }
 
                 retrieveInstanceInfo(imagePath);
                 $cqFileUpload.on("assetselected", function(e) {
                     fileReference = e.path;
                     // if it is a remote asset
-                    remoteFileReference = $cqFileUpload.find("input[name='./fileReference']").val();
-                    if (fileReference === undefined && remoteFileReference && remoteFileReference !== "" &&
-                        remoteFileReference.includes("urn:aaid:aem")) {
-                        fileReference = remoteFileReference;
-                        smartCropRenditionFromJcr = "NONE"; // for newly selected asset we clear the smartcrop selection dropdown
-                    }
-                    retrieveDAMInfo(fileReference).then(
-                        function() {
-                            if (isDecorative) {
-                                altTuple.hideCheckbox(isDecorative.checked);
-                            }
-                            captionTuple.hideCheckbox(false);
-                            altTuple.reinitCheckbox();
-                            captionTuple.reinitCheckbox();
-                            toggleAlternativeFieldsAndLink(imageFromPageImage, isDecorative);
-                            if (areDMFeaturesEnabled && !isPolarisEnabled) {
-                                selectPresetType($(presetTypeSelector), "imagePreset");
-                                resetSelectField($dynamicMediaGroup.find(smartCropRenditionDropDownSelector));
+                    if (fileReference === undefined) {
+                        var $fileReferences = $cqFileUpload.find("[data-cq-fileupload-parameter='filereference']");
+                        $fileReferences.each(function() {
+                            var remoteFileReference = $(this).val();
+                            if (remoteFileReference && remoteFileReference !== "" &&
```

Comment:
if(remoteFileReference && remoteFilreReference !== ""),  seems to me redundant since an empty string is falsy